From c29f0ce8015fa2fbd4e282738a8068c1cb2dfed3 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Thu, 31 Mar 2022 08:25:45 +0300
Subject: [PATCH 061/103] mmc: fsl_esdhc_imx: Revert HS200 & HS400 flags
 interpretation

Remove HS200 & HS400 flags intereptation as this is already part of
commit ecb8c88d800b ("mmc: fsl_esdhc_imx: Use esdhc_soc_data flags to set host caps")

Issue: ALB-8582
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/mmc/fsl_esdhc_imx.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/mmc/fsl_esdhc_imx.c b/drivers/mmc/fsl_esdhc_imx.c
index 2d32a28d4c..9e146ac7e6 100644
--- a/drivers/mmc/fsl_esdhc_imx.c
+++ b/drivers/mmc/fsl_esdhc_imx.c
@@ -1335,13 +1335,6 @@ static int fsl_esdhc_init(struct fsl_esdhc_priv *priv,
 	cfg->host_caps |= MMC_MODE_DDR_52MHz;
 #endif
 
-	if (priv->flags & ESDHC_FLAG_HS200)
-		cfg->host_caps |= MMC_MODE_HS200;
-	if (priv->flags & ESDHC_FLAG_HS400)
-		cfg->host_caps |= MMC_MODE_HS400;
-	if (priv->flags & ESDHC_FLAG_HS400_ES)
-		cfg->host_caps |= MMC_MODE_HS400_ES;
-
 	if (priv->bus_width > 0) {
 		if (priv->bus_width < 8)
 			cfg->host_caps &= ~MMC_MODE_8BIT;
-- 
2.25.1

